transports/daily/src/transport.ts

```diff
@@ -305,6 +409,12 @@ export class DailyTransport extends Transport {
     this._daily.stopLocalAudioLevelObserver();
     this._daily.stopRemoteParticipantsAudioLevelObserver();
 
+    this.audioQueue = []
+    this.currentAudioTrack = null
+    if (this.mediaStreamRecorder && this.mediaStreamRecorder.getStatus() !== "ended") {
+      void this.mediaStreamRecorder.end()
+    }
```

Comment:
could this be replaced with a call to `stopRecording()`? Is there a reason we wouldn't want to send the recording-stopped event in this case?

Reply:
Yep, it makes sense.

Reply:
Done.